README.md

```diff
@@ -517,6 +522,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Bitwarden**](https://github.com/bitwarden/mobile)
 * [**FreeOTP+**](https://github.com/helloworld1/FreeOTPPlus) <sup>**[[F-Droid](https://f-droid.org/app/org.liberty.android.freeotpplus)]**</sup>
 * [**KeePassDX**](https://github.com/Kunzisoft/KeePassDX) <sup>**[[F-Droid](https://f-droid.org/app/com.kunzisoft.keepass.libre)]**</sup>
+* [**Keepass2Android**](https://github.com/PhilippC/keepass2android)
```

Comment:
This project seems to include proprietary components (https://github.com/PhilippC/keepass2android/issues/38, https://gitlab.com/fdroid/rfp/-/issues/5, https://apt.izzysoft.de/fdroid/index/apk/keepass2android.keepass2android_nonet).

Reply:
> I don't really understand what Signal, which also has proprietary dependencies, is doing in the list.

That's a valid point, so we should reconsider Signal.

Reply:
[Done](https://github.com/offa/android-foss/pull/304/commits/412b6eabb4eb5ddf64f7e68acafc4251a18cb69c) :D